### PR TITLE
Add my-name to the list of contributors

### PR DESCRIPTION
**Pull request by Abhi6722**

### CODE REVIEW

**Review by NS007-dev — CHANGES_REQUESTED** [review]
Please format the file @Abhi6722 .
Thank you.

**Review by Abhi6722 — COMMENTED** [review]
Page Formated

File: Data/member.json [hunk 1]
```diff
@@ -500,15 +500,15 @@
...
         },
-      {
+        {
             "name": "Aleksey-Voko",
             "avatar": "https://avatars.githubusercontent.com/u/61884038?v=4",
             "description": "Python, Java, Kotlin, Android, Go",
             "githuburl": "https://github.com/Aleksey-Voko",
             "linkedinurl": ""
         }
         ,
```
Comment (NS007-dev) [inline Data/member.json]: Please can you move the comma to be on line 509, and after that it should be fine @Abhi6722

**Review by NS007-dev — CHANGES_REQUESTED** [review]

File: Data/member.json (hunk 1 above)
Comment (Abhi6722) [reply Data/member.json]: Done @CodingSpecies

**Review by NS007-dev — APPROVED** [review]

**Review by MrSlimCoder — APPROVED** [review]
LGTM

**Comment by MrSlimCoder** [comment]
@all-contributors add @Abhi6722 for content.

**Pull request merged**